src/view/notification.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { View, StyleSheet } from 'react-native';
+import { observer } from 'mobx-react';
+import PropTypes from 'prop-types';
+import Background from '../component/background';
+import { Header, Title } from '../component/header';
+import { Button, BackButton, SmallPillButton } from '../component/button';
+import { ListContent, List, ListItem, ListHeader } from '../component/list';
+import { Alert } from '../component/notification';
+import Text from '../component/text';
+import { color, font } from '../component/style';
+
+//
+// Notification View
+//
+
+const NotificationView = ({ store, nav }) => {
+  const { computedNotifications: notifications } = store;
+  return (
+    <Background color={color.blackDark}>
+      <Header separator>
+        <BackButton onPress={() => nav.goSettings()} />
+        <Title title="Notifications" />
+        <Button disabled onPress={() => {}} />
+      </Header>
+      <ListContent>
+        <List
+          data={notifications}
+          renderHeader={() => <NotificationListHeader />}
+          renderItem={item => <NotificationListItem item={item} />}
+        />
+      </ListContent>
+    </Background>
+  );
+};
+
+NotificationView.propTypes = {
+  store: PropTypes.object.isRequired,
+  nav: PropTypes.object.isRequired,
+};
+
+//
+// Notification List Item
+//
+
+const iStyles = StyleSheet.create({
+  wrap: {
+    flex: 1,
+    paddingRight: 50,
+  },
+  txt: {
+    color: color.white,
+    fontSize: font.sizeS,
+  },
+  alert: {
+    marginRight: 6,
+  },
+  group: {
+    flexDirection: 'row',
+    alignItems: 'center',
+  },
+  l: { flex: 8 },
```

Comment:
i'm a little confused by these variable names

Reply:
Just t-shirt sizes to set relative column width :)